i6_models/parts/blstm.py

```diff
@@ -0,0 +1,78 @@
+from dataclasses import dataclass
+import torch
+from torch import nn
+
+from i6_models.config import ModelConfiguration
+
+
+@dataclass()
+class BlstmEncoderV1Config(ModelConfiguration):
+    """
+    Attributes:
+        num_layers: number of bi-directional LSTM layers, minimum 2
+        input_dim: input dimension size
+        hidden_dim: hidden dimension of one direction of LSTM, the total output size is twice of this
+        dropout: nn.LSTM supports internal Dropout applied between each layer of BLSTM (but not on input/output)
+        enforce_sorted: keep activated for ONNX-Export, requires that the lengths are sorted decreasing from longest
```

Comment:
I still think this description is a bit misleading or incomplete. `enforce_sorted=True` actually means that the model expects that the sequences are sorted, `enforce_sorted=False` means that it will enforce internally that sequences are sorted and it is not needed that they are sorted beforehand.

```suggestion
        enforce_sorted:
            True: expects that sequences are sorted by sequence length in decreasing order.
                Will not do any sorting.
                This is required for ONNX-Export.
            False: no expectation.
                It will internally enforce that they are sorted
                and undo the reordering at the output.        
```